shopfloor/actions/message.py

```diff
@@ -199,11 +199,16 @@ def no_product_for_barcode(self, barcode):
             "body": _("No product found for {}").format(barcode),
         }
 
-    def wrong_product(self):
```

Comment:
this way you break https://github.com/OCA/wms/blob/b84a3ef9d4d3f9667707316a1fb7afa76a8dc796/shopfloor_manual_product_transfer/services/manual_product_transfer.py#L483

Please:
1. do not remove this method
2. make it use the new one.
3. log a deprecation warning if we expect to drop it completely for v15
4. adapt shopfloor_manual_product_transfer to use the new method

Reply:
Thanks, done